packages/tooling/__tests__/lib/parameters-to-json-schema.test.js

```diff
@@ -429,37 +416,21 @@ describe('type', () => {
         },
       ];
 
-      expect(parametersToJsonSchema({ parameters })[0].schema).toStrictEqual({
-        properties: { param: { items: {}, type: 'array' } },
-        required: [],
-        type: 'object',
-      });
-    });
-
-    it('should repair a malformed object that is typod as an array [README-6R]', () => {
```

Comment:
`@openapi-contrib/openapi-schema-to-json-schema` ignores `properties` here because it's an array so this is no longer a case we need to try to repair.